src/partials/70_section-blog-posts.html

```diff
@@ -16,7 +16,7 @@
     <div class="row">
       <div class="col-4">
         <div class="post-box">
-          <div class="cover">
+          <div class="cover post1">
```

Comment:
Bardzo zła praktyka. Zastanów się jeśli byśmu mieli brać dane o postach z bazy danych to jak przekażesz do niego obrazek. Czy dla każdego będziesz generować klasę w css? Pewnie nie.

Dlatego lepszą praktyką jeat dodanie go jako inline styles

```suggestion
          <div class="cover" style="background-image: url(../images/post1.jpeg);">
```